src/translator.py

```diff
@@ -32,91 +34,205 @@ def __init__(self, dictionary_description: str, translator_description: str):
 
 
 class Languages(Enum):
+    """
+    Class representing languages using enum module.
+    These are not mutable = once defined, these won't be changed.
+    """
     ENGLISH = auto()
     POLISH = auto()
     SPANISH = auto()
     GERMAN = auto()
+    ITALIAN = auto()
+    FRENCH = auto()
 
 
 class Dictionary:
+    """
+    Class representing a dictionary from one language to another.
+    Each Dictionary is initialized with private attributes.
+    Use setters & getters to interact with them instead of accessing them directly.
+    Each Dictionary comes with an empty dictionary of translations.
+    Use the appropriate method to add new translations to this dictionary.
+    """
 
     @staticmethod
     def get_dictionary_description(from_lang: Languages, to_lang: Languages):
+        """
+        Method to return description of the Dictionary = the languages pair
+        It's a @staticmethod as it can't change object instance state nor class state,
+        this functionality is related to the class, but it doesn't need to access nor modify the class or its instances.
+        """
+
         return f"{from_lang.name.capitalize()} -> {to_lang.name.capitalize()} dictionary"
 
     def __init__(self, from_lang: Languages, to_lang: Languages):
         self._from_lang = from_lang
         self._to_lang = to_lang
-        self._translations = dict()
+        self._translations = {}
 
     @property
     def languages(self):
+        """
+        Method to get the languages pair of this Dictionary.
+        This is a 'getter' method and a property of each dictionary initialized from this class.
+        """
         return self._from_lang, self._to_lang
 
+    @property
+    def all_translations(self):
+        return self._translations
+
     def __str__(self):
         return self.get_dictionary_description(self._from_lang, self._to_lang)
 
+    def import_translations(self, filename: str):
+        """
+        The file must be of .JSON format.
+        The .JSON format doesn't support sets - translations must be arrays, e.g.:
+        "zamek": ["castle", "lock"],
+        "pies": ["dog"]
+        """
+        self._filename = filename
+
+        with suppress(FileNotFoundError):

```

Comment:
No need to use suppress here. Just allow the FileNotFoundError to be raised.

Reply:
I've changed that in my recent commit.